### DISCUSSION TITLE
Missing response headers in ServerFilter

### DISCUSSION THREAD

**Discussion by s-af** [micronaut-projects/micronaut-core#10127] [Q&A]
Hi.
I would like to ask if there are any options how to get all response headers inside (a)ServerFilter or legacy HttpServerFilter. 
In my test examples I was able to read only "date" header. "Content-Type" and "content-length" are always missing. I was playing around with (a)ServerFilter, legacy HttpServerFilter and different Orders for them with no success. 
There is example:
```
@Slf4j
@ServerFilter(RestController.PATH)
public class RestControllerFilter {

    @Order(Integer.MIN_VALUE)
    @ResponseFilter
    void filter(
            HttpResponse<?> response
    ) {
        log.info("Response headers: {}", response.getHeaders().asMap());
    }
}
``` 
There is console output:
``` 
Response headers: {date=[Thu, 16 Nov 2023 10:08:39 GMT]}
``` 
And curl output:
``` 
< HTTP/1.1 200 OK
< date: Thu, 16 Nov 2023 10:08:39 GMT
< Content-Type: text/plain
< content-length: 2
<
```

**Comment by yawkat** [comment]
no it's not currently possible
